sql-and-jdbc/src/main/java/org/example/AddPerson.java

```diff
@@ -0,0 +1,38 @@
+package org.example;
+
+import java.sql.Date;
+import java.sql.SQLException;
+import java.util.Scanner;
+
+public class AddPerson {
```

Comment:
This is a very interesting way to approach it. You have encapsulated different types of actions into separate classes - which you means you have basically invented the Command design pattern. Good job!